test_app/lib/widgets/text_form_field.dart

```diff
@@ -51,9 +55,20 @@ class _InputTextFormFieldState extends State<InputTextFormField> {
           borderRadius: const BorderRadius.all(Radius.circular(12.0)),
           borderSide: BorderSide(color: widget.focusColor, width: 2),
         ),
-        suffixIcon: IconButton(
-            onPressed: widget.controller!.clear,
-            icon: const Icon(Icons.clear, color: Colors.grey)),
+        suffixIcon: Row(
+            mainAxisAlignment: MainAxisAlignment.end,
+            mainAxisSize: MainAxisSize.min,
+            children: [
+              widget.isStorageNeeded
+                  ? IconButton(
+                      onPressed: widget.onStorageIconTap,
+                      icon: const Icon(Icons.storage),
+                      color: Colors.grey)
+                  : const SizedBox.shrink(),
+              IconButton(
+                  onPressed: widget.controller!.clear,
+                  icon: const Icon(Icons.clear, color: Colors.grey))
+            ]),
```

Comment:
Formatting
```suggestion
        suffixIcon: Row(
          mainAxisAlignment: MainAxisAlignment.end,
          mainAxisSize: MainAxisSize.min,
          children: [
            widget.isStorageNeeded
                ? IconButton(
                    onPressed: widget.onStorageIconTap,
                    icon: const Icon(Icons.storage),
                    color: Colors.grey,
                  )
                : const SizedBox.shrink(),
            IconButton(
              onPressed: widget.controller!.clear,
              icon: const Icon(Icons.clear, color: Colors.grey),
            )
          ],
        ),
```

Reply:
Forgot about it. I'll apply this. Thanks